tools/harness-thci/OpenThread.py

```diff
@@ -3111,6 +3139,14 @@ def role_transition(self, role):
     def setLeaderWeight(self, iWeight=72):
         self.__executeCommand('leaderweight %d' % iWeight)
 
+    @watched
+    def isBorderRoutingEnabled(self):
+        try:
+            self.__executeCommand('br omrprefix')
```

Comment:
add note for the trick?

Reply:
The code should be self-explanatory? 
Basically, we run a `BORDER_ROUTING` command to check if the `BORDER_ROUTING` feature is enabled or not.
The same technique is often used by OTCI to check whether certain feature is enabled.

Reply:
Will keep the code as is since the method name `isBorderRoutingEnabled` already expains what the method is doing.